src/Features/Core/Portable/GenerateMember/GenerateConstructor/AbstractGenerateConstructorService.Editor.cs

```diff
@@ -356,11 +370,11 @@ private async Task<(Document, bool addedFields)> GenerateFieldDelegatingConstruc
                                 // Uh-oh.  Now we have a problem.  We can't assign this parameter to
                                 // this field.  So we need to create a new field.  Find a name not in
                                 // use so we can assign to that.  
-                                var newFieldName = NameGenerator.EnsureUniqueness(
-                                    attributeArguments != null
-                                        ? _service.GenerateNameForArgument(_document.SemanticModel, attributeArguments.Value[index], _cancellationToken)
-                                        : _service.GenerateNameForArgument(_document.SemanticModel, arguments[index], _cancellationToken),
-                                    GetUnavailableMemberNames().Concat(parameterToNewFieldMap.Values));
+                                var baseName = attributeArguments != null
+                                ? _service.GenerateNameForArgument(_document.SemanticModel, attributeArguments.Value[index], _cancellationToken)
+                                : _service.GenerateNameForArgument(_document.SemanticModel, arguments[index], _cancellationToken);
```

Comment:
💡 Incorrect indentation of ternary expression (continuation lines need to be indented relative to `var`)

Reply:
Fixed